repository-data/development/src/main/resources/hcm-actions.yaml

```diff
@@ -67,3 +67,7 @@ action-lists:
       /content/documents/lks/landing-pages: reload
       /content/documents/lks/cookies: reload
       /content/documents/lks/listing-pages: reload
+  - 1.0.35:
+      /content/documents/lks/listing-pages: reload
+      /content/documents/lks/news: reload
```

Comment:
Since `/content/documents/lks/news` is a brand new folder, we don't necessarily need to reload.

Reply:
Done.